.github/workflows/calypso-live.yml

```diff
@@ -0,0 +1,45 @@
+name: Calypso Live
+
+on:
+  pull_request:
+    types: ['opened']
+
+jobs:
+  calypso-live:
+    name: 'Launch a Calypso.live instance for your branch'
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
+    timeout-minutes: 10
+
+    steps:
+      - name: Extract branch name
+        run: |
+          echo ::set-output name=BRANCH::${{ github.event.pull_request.head.ref }}
+        id: extract_branch
+
+      - name: Build Calypso.live link.
+        run: |
+          echo '::set-output name=LINK::https://calypso.live/?branch=${{ steps.extract_branch.outputs.BRANCH }}'
+        id: build_link
+
+      - name: Post comment on PR
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'Test live: ${{ steps.build_link.outputs.LINK }}'
+            })
+
+      - name: Fetch link
```

Comment:
Testing locally, it looks like `calypso.live` always responds with 200, even when the branch doesn't exist (`curl -o /dev/null -s -w "%{http_code}\n" "https://calypso.live?branch=does-not-exist-qwerty"`). If that's the case, maybe we should remove this step

(I assume that's what this is testing -- is it something else?)

Reply:
I've based this off the existing implementation in the webhook, where we do:

```php
// Boot the branch.
wp_remote_get( 'https://calypso.live/?branch=' . $branch, array(
	'blocking' => false,
	'timeout'  => 120
) );
```

I assume that's to get the build process started before the person clicks on the link?

Reply:
ahh, makes sense 👍 

Could we add a comment mentioning that here too?

Reply:
Sure thing, you're right. Added in comment in 9143ee4484d5f473d4fea1fb588bd5c42cd9e3ab